src/optimal_planner.cpp

```diff
@@ -127,6 +127,7 @@ void TebOptimalPlanner::registerG2OTypes()
   factory->registerType("EDGE_SHORTEST_PATH", new g2o::HyperGraphElementCreator<EdgeShortestPath>);
   factory->registerType("EDGE_VELOCITY", new g2o::HyperGraphElementCreator<EdgeVelocity>);
   factory->registerType("EDGE_VELOCITY_HOLONOMIC", new g2o::HyperGraphElementCreator<EdgeVelocityHolonomic>);
+  factory->registerType("EDGE_VELOCITY_ABSOLUTE", new g2o::HyperGraphElementCreator<EdgeVelocityAbsolute>);
```

Comment:
where is the other one?

Reply:
Good catch. Internally not needed as the factory is not used. Will add it.